runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -230,11 +230,11 @@ def SanityCheck(): bool
     err = 'Cannot debug, +channel feature is not supported'
   elseif (way is Way.Prompt) && !exists('*prompt_setprompt')
     err = 'Cannot debug, missing prompt buffer support'
-  elseif (way is Way.Prompt) && !empty(glob(gdb_cmd))
+  elseif (way is Way.Prompt) && !empty(glob($'./{gdb_cmd}')) || !empty(glob($'./{gdb_cmd}/'))
     err = $"You have a file/folder named '{gdb_cmd}' in the current directory Termdebug may not work properly. Please exit and rename such a file/folder."
-  elseif !empty(glob(asmbufname))
+  elseif !empty(glob($'./{asmbufname}')) || !empty(glob($'./{asmbufname}/'))
     err = $"You have a file/folder named '{asmbufname}' in the current directory Termdebug may not work properly. Please exit and rename such a file/folder."
-  elseif !empty(glob(varbufname))
+  elseif !empty(glob($'./{varbufname}')) || !empty(glob($'./{varbufname}/'))
```

Comment:
I don't think we need duplicating the `glob()` argument. If there exists a file, then you cannot have a directory with the same name (at least not usually). Even if it was possible, I think adding the explicit path separator at the end would not make a match, in other words I think `glob('foobar')` would match the same directory as `glob('foobar/')` would

Reply:
I'll try it! :) Thanks for the feedback!